dataprofiler/tests/profilers/test_base_column_profilers.py

```diff
@@ -165,7 +163,7 @@ def test_combine_unqiue_sets(self):
         a = [1, 2, 3]
         b = [3, 1, 4, -1]
         c = utils._combine_unique_sets(a, b)
-        six.assertCountEqual(self, [1, 2, 3, 4, -1], c)
+        self.assertCountEqual([1, 2, 3, 4, -1], c)
```

Comment:
still has `import six` in the file

Reply:
`autoflake` by default only catches unused imports that come from standardlib. Out of the box cleaning up all seemingly unused imports can be risky because they may be relied on as side effects. `autoflake` can be used with the following settings to adjust the default behavior:
```
 --imports IMPORTS     by default, only unused standard library imports are removed; specify a comma-separated list of additional modules/packages
  --expand-star-imports
                        expand wildcard star imports with undefined names; this only triggers if there is only one star import in the file; this is skipped if there are any uses of `__all__` or `del` in the file
  --remove-all-unused-imports
                        remove all unused imports (not just those from the standard library)
```

Reply:
i see